.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,67 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
+    branches:
+      - 'main'
+    paths:
+      - 'taqueria-plugin-*/docker/*'
+  create:
```

Comment:
I think we should use `push: tags` here like the other workflow

Reply:
:upside_down_face: Changed